clients/python/example/pingpong/Makefile

```diff
@@ -0,0 +1,6 @@
+build:
+	docker build -t gcr.io/insprlabs/inspr/example/python/ping:latest -f ping/ping.Dockerfile ../..
+	docker push gcr.io/insprlabs/inspr/example/python/ping:latest
+
+	docker build -t gcr.io/insprlabs/inspr/example/python/pong:latest -f pong/pong.Dockerfile ../..
+	docker push gcr.io/insprlabs/inspr/example/python/pong:latest
```

Comment:
commentary of similar nature of the one done in the ping.Dockerfile, no need to pass the context `../..` to the `docker build` command, since it only access the ping file that is below it,

If you decided to do a `COPY ping.py .` in the ping.Dockerfile, you would only need the context of `./ping` in your Makefile.